SEL32/sel32_chan.c

```diff
@@ -80,12 +80,13 @@
 
 uint32  channels        = MAX_CHAN;             /* maximum number of channels */
 int     subchannels     = SUB_CHANS;            /* maximum number of subchannel devices */
-int     irq_pend        = 0;                    /* pending interrupt flag */
+//int     irq_pend        = 0;                    /* pending interrupt flag */

```

Comment:
If you want code to go away, please delete it.  Commented out code (or ifdef-ed out) is for debugging, it shouldn't be in completed code.

Reply:
These are comments to me about things I change during testing.  They usually go away during the next cleanup cycle.